taipy/core/data/_data_manager.py

```diff
@@ -38,11 +39,33 @@ def _get_or_create(
         scope = data_node_config.scope
```

Comment:
Is the method _get_or_create still used?

Reply:
At least in test. We have to check for: datanode, task and pipeline.

Reply:
SHould not we use the `bulk_get_or_create` So we are using the real-life code?